### ISSUE TITLE
Cant clone git repo from other docker container

### ISSUE THREAD

**Issue by jlndk** [concourse/concourse#922]
Hi Everyone!

I'm trying to setup concourse together with my bitbucket (git) server.
I've managed to get them running together, using docker compose and a [nginx proxy](https://github.com/jwilder/nginx-proxy), however I cant get concourse to clone my repo.

When i start a new build in concourse i get the following error in my git resource:
```
resource script '/opt/resource/check []' failed: exit status 128

stderr:
Cloning into '/tmp/git-resource-repo-cache'...
fatal: unable to access 'http://bitbucket:7990/scm/proj/server.git/': Couldn't resolve host 'bitbucket'
```

Since docker-compose creates an alias for each container and connects them on the same virtual network i should be able to refer to the bitbucket server by the alias.

This is my current pipeline:
```yml
resources:
- name: git-server
  type: git
  source:
    uri: http://bitbucket:7990/scm/proj/server.git
    username: concourse
    branch: production
    password: {{concourse-git-password}}

jobs:
- name: test-laravel
  public: true
  plan:
  - get: git-server
  - task: phpunit
    config:
      platform: linux
      image_resource:
        type: docker-image
        source: {repository: phpunit/phpunit}
      run:
        path: phpunit
```

And my `docker-compose.yml`
```yml

nginx-proxy:
    image: jwilder/nginx-proxy
    ports:
      - "80:80"
    volumes:
      - /var/run/docker.sock:/tmp/docker.sock:ro

#
# BITBUCKET
# This container acts as our code repository and provides a place for us
# to manage our code.
#
bitbucket:
    image: atlassian/bitbucket-server
    links:
       #Link to a database container
        - bitbucket_data
    volumes:
        - ./data/bitbucket/application-data:/var/atlassian/application-data/bitbucket
    expose:
        - "7990"
    environment:
        - VIRTUAL_HOST=dev.domain.local
        - VIRTUAL_PORT=7990
        #Tell bitbucket about our nginx proxy
        - CATALINA_CONNECTOR_PROXYNAME=dev.domain.local
        - CATALINA_CONNECTOR_PROXYPORT=80

###Definition for the bitbucket database......


concourse-web:
    image: concourse/concourse
    links: [concourse-db]
    command: web
    volumes: ["./data/concourse/keys/web:/concourse-keys"]
    expose:
        - "8080"
    environment:
        VIRTUAL_HOST: "ci.domain.local"
        VIRTUAL_PORT: "8080"
        CONCOURSE_BASIC_AUTH_USERNAME: concourse
        CONCOURSE_BASIC_AUTH_PASSWORD: myconcoursepassword
        CONCOURSE_EXTERNAL_URL: ""
        CONCOURSE_POSTGRES_DATA_SOURCE: |-
            postgres://concourse:myconcoursepassword@concourse-db:5432/concourse?sslmode=disable

##Definition for the concourse database...

concourse-worker:
    image: concourse/concourse
    privileged: true
    links: [concourse-web]
    command: worker
    volumes: ["./data/concourse/keys/worker:/concourse-keys"]
    environment:
        CONCOURSE_TSA_HOST: concourse-web
```

Other details: 
* Concourse version: 2.7.0 (latest tag on docker hub)
* Deployment type: Docker compose
* Did this used to work?: No

**Comment by chendrix** [comment]
@jlndk I believe this is the same as #1145 ?

**Comment by andrewedstrom** [comment]
Closing since, as @chendrix suggested, this seems to be a duplicate. If this was not solved by #1145 please reopen.
